scripts/test-fuzzed-MAC.py

```diff
@@ -47,12 +51,13 @@ def main():
         node = node.add_child(FinishedGenerator())
         node = node.add_child(ExpectChangeCipherSpec())
         node = node.add_child(ExpectFinished())
-        node = node.add_child(fuzz_mac(ApplicationDataGenerator(
-                                                        b"GET / HTTP/1.0\n\n"),
-                                       xors={pos:val}))
+        node = node.add_child( \
+            fuzz_seal(ApplicationDataGenerator( b"GET / HTTP/1.0\n\n"), xors={pos:val})
```

Comment:
> fuzz_message() not used due to error TypeError: unsupported operand type(s) for ^=: 'str' and 'int' when it attempts to apply xor to a string.

that's weird... `scripts/test-fuzzed-finished.py` uses `fuzz_message` and it works just fine in both Py2 and Py3

> I can try develop a fuzz_ciphertext() as you suggested if this is the best way forward.

it allows for arbitrary fuzzing of unencrypted messages and kills two birds with one stone (fuzzing EtM and AEAD)